resources/developer-mode.mjs

```diff
@@ -101,11 +102,21 @@ function createUIForSyncStepDelay() {
     return label;
 }
 
-function createTimeRangeUI(labelText, initialValue, unit = "ms", min = 0, max = 1000) {
+function createUIForComplexity() {
+    const { range, label } = createTimeRangeUI("Relative complexity", params.complexity, "x", 0, 10, 0.01);
```

Comment:
I was thinking that rather than a float, maybe this can be an integer between 1 and 100, with 50 being the default? 

Then later you can divide this number by 50 to get the multiplier.
And you can cut down a lot of the changes.

But this is merely a suggestion, I'm also happy with what you did.

Reply:
I kinda prefer a normalized float value, aiming quite literaly at a complexity-factor, 2x => it should run roughly twice as slow.